lib/transaction/input/input.js

```diff
@@ -158,7 +158,7 @@ Input.prototype.isFullySigned = function() {
 };
 
 Input.prototype.isFinal = function() {
-  return this.sequenceNumber !== 4294967295;
+  return this.sequenceNumber === 4294967295; //0xffffffff
```

Comment:
We already have MAXINT defined as a constant at the top of the file. Let's use that instead.